### ISSUE TITLE
Thelio case needs indentations or handles for lifting

### ISSUE THREAD

**Issue by eric-s-raymond** [system76/thelio#8]
Title says it all.

The nice custom Unix epoch zero fan grille on mine is slightly bent inwards because the top of it was the only place my fingers could get purchase when I was trying to shift the thing. I am sad about this.

I don't much care functionally whether the grips or indented portions are fore and aft near the top  of either end of the lower frame or on either side of the case top.  The fore-and-aft placement would probably be better structural engineering, as it would avoid hanging the weight on the lower frame off of six relatively small thumbscrews.

**Comment by WatchMkr** [comment]
Thank you for the suggestion. We'll keep this in mind as we continue to iterate the design.

**Comment by magdesign** [comment]
May sound a bit silly but, 
look out for some old MacPro mid2010 on ebay.
Their case is beautifully designed, very easy to open, everything exchangeable without tools, very silent and they are not in production anymore.... just for inspiration..

**Comment by eric-s-raymond** [comment]
Yeah, those middle-period Mac cases were excellent design for usability/maintainability.

If you want real positive differentiation this is the direction I urge you to go - make people think "motherboards age out but I want to keep this case forever".  My old NZXT case is like that.
